Dockerfile-e2e

```diff
@@ -1,8 +1,18 @@
-FROM cypress/browsers:node18.12.0-chrome107
+FROM 113355358853.dkr.ecr.eu-west-1.amazonaws.com/ho/hmpo/lev/fro/cypress-testing:8c64f6e43a7e98ec1a25a1bf2bf7d830ef151e56
+# Use image below for local testing. Proxies also require removal.
```

Comment:
I think it might be worth adding some of these instructions (including the commented images in the docker-compose files) into the readme.

Reply:
I have updated the README with some info about local testing